### PR TITLE
Russian i18n

### PR DESCRIPTION
**Pull request by Rurik19**
@Simounet please look at this PR, there are changes in the code.

### CODE REVIEW

File: wallabagger/js/common.js [hunk 1]
```diff
@@ -16,8 +16,11 @@ const Common = (() => {
         });
     };
 
+    const getLocale = () => browser.i18n.getUILanguage();
+
     return {
         'translate': translate,
-        'translateAll': translateAll
-    }
+        'translateAll': translateAll,
```
Comment (tcitworld) [inline wallabagger/js/common.js]: Couldn't you just use shorthand properties here ?

**Comment by Simounet** [comment]
Hi @Rurik19 , I made some changes. Can you try it, please?

File: wallabagger/js/common.js (hunk 1 above)
Comment (Rurik19) [reply wallabagger/js/common.js]: yeah. that would be more clear, but let's change it id Dev branch after this PR

**Comment by Rurik19** [comment]
It seems alright

**Pull request merged**